packages/account/src/Styles/account.scss

```diff
@@ -2680,4 +2680,236 @@ $MIN_HEIGHT_FLOATING: calc(
+                    .dc-input {
+                        width: unset;
+                    }
+                }
+                @include mobile {
+                    flex-direction: column;
+                    padding-bottom: 4rem;
+                }
+
+                &-cardnumber {
+                    border: 1px solid #d6dadb;
+                    box-sizing: border-box;
+                    border-radius: 4px;
+                    height: 4rem;
+                    margin-right: 10px;
+                    color: var(--text-general);
+                    -webkit-text-fill-color: var(--text-general);
+                    @include mobile {
+                        width: 100%;
+                    }
+                }
+
+                &-photo {
+                    margin-bottom: 35px;
+                    @include mobile {
+                        width: 100%;
+                        margin-left: 0;
+                        margin-top: 1.6rem;
+                    }
+                    @include desktop {
+                        .dc-input {
+                            flex-grow: 8;
+                        }
+                    }
+                    &-btn {
+                        margin-left: 1rem;
+                        padding: 10px 16px;
+                        position: static;
+                        height: 40px;
+                    }
+                }
+            }
+        }
+    }
+    &__form {
+        &-container {
+            padding-top: 16px;
+        }
+
+        &-content {
+            display: flex;
+            overflow: visible;
+        }
+    }
+    &__progress {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        margin-right: 25px;
+        overflow: visible;
+        @include mobile {
+            margin-right: 8px;
+        }
+        &-number {
+            min-width: 24px;
+            max-width: 24px;
+            min-height: 24px;
+            max-height: 24px;
+            border-radius: 50px;
+            background-color: var(--brand-red-coral);
+            color: var(--state-normal);
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            font-size: 16px;
+            font-weight: bold;
+        }
+        &-bar {
+            width: 2px;
+            height: 120%;
+            background-color: var(--brand-red-coral);
+            position: relative;
+        }
+    }
+    &__container {
+        text-align: center;
+        padding: 2rem auto;
+        width: 100%;
+        > * {
+            text-align: center;
+            display: block;
+            margin: 2rem auto;
+            text-align: center;
+        }
+    }
+    &-link {
+        font-weight: 400;
+        font-size: 1.4rem;
+    }
+    &-valid-identifier {
+        -webkit-text-fill-color: $color-green-1 !important;
+        border-color: $color-green-1 !important;
+    }
+}
+.sample_credit_card_modal {
```

Comment:
@jim-deriv please change this classname to kebab-case for consistency and following BEM methodology  🙂

Reply:
I'll update it @maryia-binary Thank you